### PR TITLE
Pladaria/dev

### PR DESCRIPTION
**Pull request by pladaria**
add share support

### CODE REVIEW

File: src/__tests__/utils-test.ts [hunk 1]
```diff
@@ -41,6 +42,30 @@ test('attach to email', async cb => {
     });
 });
 
+test('share', async cb => {
```
Comment (atabel) [inline src/__tests__/utils-test.ts]: why async?

File: src/webview-bridge-cjs.js.flow [hunk 2]
```diff
@@ -97,6 +97,17 @@ declare export function attachToEmail({|
     body?: string,
 |}): Promise<void>;
 
+export type ShareOptions =
+    | {
```
Comment (atabel) [inline src/webview-bridge-cjs.js.flow]: exact?

File: src/__tests__/utils-test.ts (hunk 1 above)
Comment (pladaria) [reply src/__tests__/utils-test.ts]: leftovers...

File: src/webview-bridge-cjs.js.flow (hunk 2 above)
Comment (pladaria) [reply src/webview-bridge-cjs.js.flow]: sure!

File: src/__tests__/utils-test.ts (hunk 1 above)
Comment (pladaria) [reply src/__tests__/utils-test.ts]: fixed

File: src/webview-bridge-cjs.js.flow (hunk 2 above)
Comment (pladaria) [reply src/webview-bridge-cjs.js.flow]: fixed

**Pull request merged**